Cargo.toml

```diff
@@ -28,10 +28,10 @@ url = "2.1.0"
 uuid = { version = "0.8.1", default-features = false, features = ["serde"] }
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-hyper = { version = "0.14.1", default-features = false, features = ["tcp", "client", "http1", "http2"] }
+hyper = { version = "0.14.1", default-features = false, features = ["tcp", "client", "http2"] }
 
 [target.'cfg(all(any(target_os = "linux", target_family = "windows", target_os = "macos"), any(target_arch = "x86", target_arch = "x86_64", target_arch = "arm", target_arch = "aarch64")))'.dependencies]
-hyper-rustls = "0.23.0"
+hyper-rustls = { version = "0.23.0", default-features = false, features = ["native-tokio", "http2", "tls12", "logging"] }
```

Comment:
Why is logging required?

Reply:
it isn't, it's one of the defaults so I thought I'd include it

Reply:
In general we'd want to disable as many defaults as possible in order to not force any upon our dependents.